format_spec/FORMAT_SPEC.md

```diff
@@ -4,7 +4,7 @@ title: Format Specification
 
 **Notes:**
 
-* The current TileDB format version number is **20** (`uint32_t`).
+* The current TileDB format version number is **21** (`uint32_t`).
```

Comment:
What changed in the storage format?

Reply:
We now generate min/max/null count correctly for fixed size nullable strings in dense arrays.

Reply:
Would an earlier Core version have trouble reading it?

Reply:
Never mind, we need the version bump to know whether we can rely on the tile metadata.